docs/guides/integration-prebid-server-side.md

```diff
@@ -0,0 +1,417 @@
+| name | Required | String | ID value for the UID2 module. Always `"uid2"`. | `"uid2"` |
+| value | Optional, server-only mode | Object | An object containing the value for the advertising token. | See [Configuration Parameter Examples: Value](#configuration-parameter-examples-value). |
+| params.uid2Token | Optional, client refresh mode | Object | The initial UID2 token. This should be the `body` element of the decrypted response from a call to the `/token/generate` or `/token/refresh` endpoint. | See [Sample Token](#sample-token). |
+| params.uid2Cookie | Optional, client refresh mode | String | The name of a cookie that holds the initial UID2 token, set by the server. The cookie should contain JSON in the same format as the uid2Token param. **If uid2Token is supplied, this param is ignored.** | See [Sample Token](#sample-token). |
+| params.uid2ApiBase | Optional, client refresh mode | String | Overrides the default UID2 API endpoint. | `"https://prod.uidapi.com"` (the default)|
+| params.storage | Optional, client refresh mode | String | Specify the module internal storage method: `cookie` or `localStorage`. We recommend that you do not provide this parameter. Instead, allow the module to use the default. | `localStorage` (the default) |
+
+### Configuration Parameter Examples: Value
+
+The following code snippet shows an example of the `value` UID2 configuration parameter.
+
+```js
+pbjs.setConfig({
+    userSync: {
+        userIds: [{
+            name: 'uid2'
+            value: {
+                'uid2': {
+                    'id': '...advertising token...'
+                }
+            }
+        }]
+    }
+});
+```
+
+### Sample Token
+
+The following sample is fictitious, but shows what the token response object, returned from either the [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) or [POST&nbsp;/token/refresh](../endpoints/post-token-refresh.md) endpoints, looks like:
+
+```js
+{
+  "advertising_token": "...",
+  "refresh_token": "...",
+  "identity_expires": 1633643601000,
+  "refresh_from": 1633643001000,
+  "refresh_expires": 1636322000000,
+  "refresh_response_key": "wR5t6HKMfJ2r4J7fEGX9Gw=="
+}
+```
+
+## Optional: Reduce Latency by Setting the API Base URL
+<!-- GWH "Reduce Latency by Setting the API Base URL" section is identical for client side and server side. -->
+By default, the UID2 module makes API calls to a UID2 server in the USA. Depending on where your users are based, you might consider choosing a server closer to your users to reduce latency.
+
+To specify a different UID2 server when you're configuring the UID2 module, set the optional params.uid2ApiBase parameter, as shown in the following example:
+
+```
+pbjs.setConfig({ 
+  userSync: { 
+    userIds: [{ 
+      name: 'uid2', 
+      params: { 
+        uid2ApiBase: baseUrl, 
+        // ... 
+      } 
+    }] 
+  } 
+}); 
+```
+
+For the list of possible base URLs, see [Environments](../getting-started/gs-environments.md).
```

Comment:
can you add the word _Integration_ into the _Testing_ row in this environment page? everywhere else we mentioned _Integration env_ but in this page we just say _Testing_

Reply:
@sunnywu good point! Done, thx.